sam/slack.py

```diff
@@ -56,9 +56,9 @@ async def get_bot_user_id():
 async def handle_message(event: {str, Any}, say: AsyncSay):
     """Handle a message event from Slack."""
     logger.debug(f"handle_message={json.dumps(event)}")
-    if event.get("subtype") == "message_deleted":
-        logger.debug("Ignoring message_deleted event %s", event)
-        return  # https://api.slack.com/events/message#hidden_subtypes
+    if event.get("subtype") in ["message_changed", "message_deleted"]:
+        logger.debug(f"Ignoring `{event['subtype']}` event {event}")
```

Comment:
@amureki loggers and f-strings don't go well together

Reply:
Well, I mindlessly followed the same style as one line above - also with `f-strings` ;-)